javascript/javascript2/week2/script.js

```diff
@@ -0,0 +1,145 @@
+// Doubling of number
+
+let numbers = [1, 2, 3, 4];
+
+const doubledOddNumbers = numbers
+.filter((myNumber) => myNumber % 2 !==0)
+.map((myNumber) =>myNumber*2);
+
+console.log(`The doubled numbers are: ${doubledOddNumbers}`) //output The doubled numbers are: 2,6
+
+// Working with movies
+
+const movies = [
```

Comment:
you got js file with lot of movies in it , try to add movies.js file to your code, instead of this.